tools/SourceKit/lib/SwiftLang/SwiftLangSupport.cpp

```diff
@@ -721,9 +721,12 @@ UIdent SwiftLangSupport::getUIDForSymbol(SymbolInfo sym, bool isRef) {
   case SymbolKind::Module:
     return KindRefModule;
 
+  case SymbolKind::CommentTag:
+    return KindCommentTag;
+
   default:
     // TODO: reconsider whether having a default case is a good idea.
```

Comment:
Can remove the TODO now (it was a bad idea 😆)

Reply:
I left this TODO because we still have the `default` case.